src/test/java/sql/dataprovider/VacancySQLDataProvider.java

```diff
@@ -0,0 +1,44 @@
+package sql.dataprovider;
+
+import sql.data.Employee;
+import sql.data.JobVacancy;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class VacancySQLDataProvider extends SQLDataProvider{
+
+    public List<JobVacancy> getVacancyList(){
+
+        String query = "SELECT hs_hr_employee.emp_number, " +
+                "hs_hr_employee.emp_firstname, " +
+                "hs_hr_employee.emp_lastname, ' % ', ohrm_job_vacancy.hiring_manager_id, ohrm_job_vacancy.name " +
+                "FROM hs_hr_employee INNER JOIN ohrm_job_vacancy " +
+                "ON hs_hr_employee.emp_number=ohrm_job_vacancy.hiring_manager_id";
+
+        List<JobVacancy> vacancyList = new ArrayList<>();
+
+        try(Connection con = DriverManager.getConnection(URL, NAME, PASSWORD);
+            Statement stmt = con.createStatement();
+            ResultSet rs = stmt.executeQuery(query)) {
+            while(rs.next()) {
+                JobVacancy jobVacancy = new JobVacancy();
```

Comment:
u set only two fields but query includes a little bit more - query can be shorter